### PR TITLE
Feature/ab2 d 4829 event service integration

### PR DESCRIPTION
**Pull request by sb-dkniesteadt**
**JIRA Tickets:**

[AB2D-4829](https://jira.cms.gov/browse/AB2D-4829) - Changes The message being grabbed and sent to sqs, One that is more valuable and can be used for multiple logging types/

### What Does This PR Do?
Change messages object that the sqs liseaner and sender uses

### What Should Reviewers Watch For?
What might break. Or whats a better way of doing this.

### Security Implications

- [ ] This PR adds new software dependencies
- [ ] This PR modifies or invalidates our security controls
- [ ] This PR stores or transmits data that was not stored or transmitted before

- [ ] This PR requires additional review of its security implications for other reasons

### Submitter Checklist

- [x] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [x] Code checked for PHI/PII exposure

### CODE REVIEW

File: ab2d-events-client/src/test/java/gov/cms/ab2d/eventclient/config/SendSqsEventTest.java [hunk 1]
```diff
@@ -56,16 +64,45 @@ void testSendMessages() throws JsonProcessingException {
...
+    }
+
+    @Test
+    void testFailedMapping(CapturedOutput output) {
+        AmazonSQS amazonSQSMock = Mockito.mock(AmazonSQS.class);
+        GetQueueUrlResult queueURL = Mockito.mock(GetQueueUrlResult.class);
+
+        when(amazonSQSMock.getQueueUrl(anyString())).thenReturn(queueURL);
+        when(queueURL.getQueueUrl()).thenReturn("localhost:4321");
+        when(amazonSQSMock.sendMessage(any(SendMessageRequest.class))).thenThrow(new UnsupportedOperationException("I am a failure"));
```
Comment (sharshar-sb) [inline ab2d-events-client/src/test/java/gov/cms/ab2d/eventclient/config/SendSqsEventTest.java:87]: Such negative talk.

File: build.gradle [hunk 2]
```diff
@@ -18,13 +18,14 @@ ext {
...
 
+    // AB2D libraries
     // AB2D libraries
     fhirVersion='1.1'
     bfdVersion='1.3'
     aggregatorVersion='1.2'
     filtersVersion='1.6'
     eventLibsVersion='1.1'
-    eventClientVersion='1.1'
+    eventClientVersion='1.1.1'
```
Comment (sharshar-sb) [inline build.gradle:28]: We're only doing minor versions for the services so this is 1.2.

**Review by sharshar-sb — CHANGES_REQUESTED** [review]

File: ab2d-events-client/src/test/java/gov/cms/ab2d/eventclient/config/SendSqsEventTest.java (hunk 1 above)
Comment (sb-dkniesteadt) [reply ab2d-events-client/src/test/java/gov/cms/ab2d/eventclient/config/SendSqsEventTest.java:87]: Well it is.

**Review by sharshar-sb — APPROVED** [review]

File: build.gradle [hunk 3]
```diff
@@ -18,13 +18,14 @@ ext {
     artifactory_password = project.hasProperty('artifactory_password') ? project.artifactory_password
             : System.getenv()['ARTIFACTORY_PASSWORD']
 
+    // AB2D libraries
```
Comment (sb-keane) [inline build.gradle:21]: Is this duplicated intentionally?

Comment (sb-dkniesteadt) [reply build.gradle]: nope, good catch

**Review by sb-keane — APPROVED** [review]

**Review by sharshar-sb — APPROVED** [review]

**Pull request merged**